content/en/docs/concepts/traffic-management/index.md

```diff
@@ -254,502 +172,246 @@ spec:
-order.
-
-A best practice pattern to specify routing rules is as follows:
-
-1. Provide one or more higher priority rules that match various conditions.
-
-1. Provide a single weight-based rule with no match condition last. This rule
-   provides the weighted distribution of traffic for all other cases.
-
-#### Precedence example with 2 rules
-
-The following virtual service configuration file includes two rules. The first
-rule sends all requests for the `reviews` service that include the Foo header
-with the bar value to the `v2` subset. The second rule sends all remaining
-requests to the `v1` subset:
+In addition to using match conditions, you can distribute traffic that
+distributes a particular routing by percentage "weight", useful for A/B testing
+and canary rollouts:
 
 {{< text yaml >}}
-apiVersion: networking.istio.io/v1alpha3
-kind: VirtualService
-metadata:
-  name: reviews
 spec:
   hosts:
   - reviews
   http:
-  - match:
-    - headers:
-        Foo:
-          exact: bar
-    route:
-    - destination:
-        host: reviews
-        subset: v2
   - route:
     - destination:
         host: reviews
         subset: v1
+      weight: 75
+    - destination:
+        host: reviews
+        subset: v2
+      weight: 25
 {{< /text >}}
 
-In this example, the header-based rule has the higher priority because it comes
-first in the configuration file. If the match-based rule came second, these
-rules wouldn't work as expected. Istio would evaluate the weight-based rule
-first and route all traffic to the instances in the `v1` subset, even requests
-including the matching `Foo` header.
+You can also use routing rules to perform some actions on the traffic, for
+example:
 
-## Destination rules
+-   Append or remove headers.
+-   Rewrite the URL.
+-   Set a [retry policy](#retries) for calls to this destination.
 
-You specify the path for traffic with routing rules, and then you use
+To learn more about the actions available, see the
+[Virtual Service reference](/docs/reference/config/networking/v1alpha3/virtual-service/#HTTPRoute).
+
+## Destination rules {#destination-rules}
+
+Along with [virtual services](#virtual-services),
 [destination rules](/docs/reference/config/networking/v1alpha3/destination-rule/)
-to configure the set of policies that Envoy proxies apply to a request at a
-specific destination.
+are a key part of Istio’s traffic routing functionality. You can think of
+virtual services as how you route your traffic **to** a given destination, and
+then you use destination rules to configure what happens to traffic **for** that
+destination. Destination rules are applied after virtual service routing rules
+are evaluated, so they apply to the traffic’s "real" destination.
+
+In particular, you use destination rules to specify named service subsets, such
+as grouping all a given service’s instances by version. Defining subsets in
+destination rules lets you cleanly refer to a specific service version across
```

Comment:
This sentence seems repetitive. I read the exact same thing in the previous section.

Reply:
You did indeed because I think I accidentally copied it rather than cut and pasted it. :)